### ISSUE TITLE
No audio detected without removing second "200 Series PCH HD Audio"

### ISSUE THREAD

**Issue by dhruvit-r** [vit9696/AppleALC#284]
Hello.

AppleALC had not been able to detect my audio devices on my laptop since v1.1.4. I have found the problematic code. Removing it works for me.
Let me know if you need any more details or logs, or if you want me to submit a pull request.

**SYSTEM CONFIGURATION**
Model: Acer VX 15 (V5-591G)
CPU: Intel i5-7300HQ
GFX: Intel HD 630 + Nvidia GTX 1050 (Disabled in macOS)
Audio: ALC255

**DPCI Manager**
Audio device: 0x8086A171
Sub-device: 0x10251127
Codec ID: 0x10EC0255
Revision: 0x0002
Codec: Realtek (Unknown)

Please take a look at: https://github.com/dhruvit-r/AppleALC/commit/1379c79119f2993a812fe1366a60bfdbe4ff5ddd

**Comment by al3xtjames** [comment]
Are you using FakePCIID or something similar? 8086:A171 is not natively supported by AppleHDAController.

**Comment by dhruvit-r** [comment]
Yes, I am. I believe FakePCIID_Intel_HDMI_Audio automatically attached to 8086:a171.

**Comment by dhruvit-r** [comment]
@Arielztd You're probably posting on the wrong issue.

**Comment by Arielztd** [comment]
Hi, vit9696/AppleALC


I am not currently using FacePCIID. Audio is working and autodetects/autoselects when headphones are plugged/removed and goes between speakers and headphone jacks, but it doesn’t detect HDMI audio when the HDMI cable is connected to a TV. It works with VoodooHDA but it’s a manual selection, not automatically.

Please I would like more details and possibly a pull request. I have tried to contribute with https://github.com/Arielztd for a working T430 on Sierra and High Sierra with Intel HD4000 graphics and an AR9280 miniPCIe halfcard WiFi configuration. Everything works except HDMI audio. I am ready to publish it but need to figure out how to post/upload the ‘tree’ so others may download it and I need to get HDMI audio working first.

If I change it too VoodooHDA (2.9.1) everything works, but I think AppleALC is a better/smarter solution and should be the audio driver because of it working with AppleHDA natively and auto selecting etc.


I also have rebranded a mini-PCIe half card AR9280 to get past the ‘whitelist’ of not allowed PCIID’s in the Lenovo ThinkPad series and get native supported WiFi in macOS.



SYSTEM CONFIGURATION
Model: ThinkPad T430 (2349-B67)
CPU: Intel i5-3320M CPU @ 2.60GHz
GFX: Intel HD 4000 (Ivy Bridge) HDMI
Audio: ALC269

DPCI Manager
Audio device: 0x80861E20
Sub-device: 0x17AA21F3
Codec ID: 0x10EC0269
Revision: 0x0203
Codec: Realtek (ALC269)
Are you using FakePCIID or something similar? 8086:A171 is not natively supported by AppleHDAController.

**Comment by al3xtjames** [comment]
Again, you are probably on the wrong issue thread. This is not a support forum - this is an issue thread specifically dealing with fixing audio on unsupported 200 Series (SKL/KBL) HDA controllers, which has nothing to do with your IVB setup (nor HDMI audio).

**Comment by Arielztd** [comment]
Thanks Alex,
                Could you point me in the direction of where to look for support for HDMI audio and how to upload a tree of data, (files and directories) a folder on GitHUB from the macOS operating system.
